src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockCropDead.java

```diff
@@ -0,0 +1,127 @@
+package net.dries007.tfc.objects.blocks.agriculture;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+import javax.annotation.Nonnull;
+
+import net.minecraft.block.BlockBush;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.properties.PropertyBool;
+import net.minecraft.block.state.BlockStateContainer;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import net.minecraftforge.common.EnumPlantType;
+
+import net.dries007.tfc.api.types.ICrop;
+import net.dries007.tfc.objects.items.ItemSeedsTFC;
+
+public class BlockCropDead extends BlockBush
+{
+    /* true if the crop spawned in the wild, means it ignores growth conditions i.e. farmland */
+    public static final PropertyBool WILD = PropertyBool.create("wild");
```

Comment:
Why do dead crops need a wild condition? They shouldn't have growth checks, nor should they particularly care i.e. if the soil beneath them is not farmland, as maybe that's exactly what killed them in the first place.

Reply:
WILD condition has been removed.